kbase/narrative
# DATAUP-312 add run_app_bulk function

# Description of PR purpose/changes

This adds the run_app_bulk function to the AppManager for starting jobs from the bulk import cell. Also a bunch of tests.

This will let the bulk import cell start jobs. I don't think it's properly wired yet to receive new job information for the job startup steps, so it will probably just look frozen with a red Cancel button. That's the next step.

You can also just run this manually with:
```python
from biokbase.narrative.jobs.appmanager import AppManager
from biokbase.narrative.jobs.jobmanager import JobManager
am = AppManager()
jm = JobManager()
new_jobs = am.run_app_bulk(
   [{ app inputs copied from a bulk import cell }]
)
jm.list_jobs()
```
And that'll show the job output. Also, it'll make a `new_jobs` dict with `parent_job` and `child_jobs` keys.

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-312

# Testing Instructions
* Details for how to test the PR:
Set up and run a bulk import cell! It will look frozen at the "Cancel" stage. But it shouldn't crash! Also, if you use the jobmanager manually to run `list_jobs`, you should see the parent job and any child jobs.

# Dev Checklist:


# Updating Version and Release Notes (if applicable)
